tds/src/main/webapp/tds.css

```diff
@@ -336,6 +336,7 @@ div.button {
     display: none;
     padding: 5px 10px;
     border: 1px solid #CFCFCF;
+    min-height: 120px;
 }
```

Comment:
This helps prevent elements from shifting when tabs resize, but the only way I can think of to eliminate it entirely is fixed height and scrolling overflow, which is...an option

Reply:
Scrolling overflow, in general, makes me sad 😢